app/server/models/reward.py

```diff
@@ -0,0 +1,24 @@
+from typing import Dict
+from sqlalchemy.dialects.postgresql import JSONB
+
+from server import db
+from server.models.utils import ModelBase
+
+
+class Reward(ModelBase):
+    __tablename__ = 'reward'
+
+    tag = db.Column(db.String)
+    recipients_data = db.Column(JSONB, default={})
```

Comment:
I would prefer we not use (more) json types in the db please.

Reply:
@nolash Why not ? 
Also in this particular instance, I think I JSON is a good idea because different rewards may collect different user data. For instance, daily bonuses collect the following:
``` 
'user_ids_with_total_unique_outward_txs' : 'some_value',
`collective_unique_outward_txs`: 'some_value'
```

Maybe a reward involving chamas will collect a different format of data for instance: 
```
'chama_tx_volume': 'some_value',
'group_membership_ratio': 'some_value',
'user_ids_with_total_unique_outward_txs': 'some_value'
```